tpot/base.py

```diff
@@ -641,6 +641,47 @@ def export(self, output_file_name):
         with open(output_file_name, 'w') as output_file:
             output_file.write(export_pipeline(self._optimized_pipeline, self.operators, self._pset))
 
+    def _contains_nan(self, features):
+        """Determine if a feature set contains any missing values.
+
+        Parameters
+        ----------
+        features: array-like {n_samples, n_features}
+
+        Returns
+        -------
+        bool, True if NaN is present, False otherwise.
+        """
+        return np.isnan(np.sum(features))
+
+    def _check_dataset(self, features, classes):
+        """Check if a dataset has a valid feature set and labels.
+
+        Parameters
+        ----------
+        features: array-like {n_samples, n_features}
+            Feature matrix
+        classes: array-like {n_samples}
+            List of class labels for prediction
+
+        Returns
+        -------
+        None
+        """
+        try:
+            check_array(features, dtype=np.float64, accept_sparse='csc')
```

Comment:
`accept_sparse` should be `None` (IIRC) because we do not currently allow sparse matrices in TPOT.